modules/apps/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/npm/NPMResolverImpl.java

```diff
@@ -40,12 +36,14 @@
 public class NPMResolverImpl implements NPMResolver {
 
 	public NPMResolverImpl(
-		Bundle bundle, JSONFactory jsonFactory, NPMRegistry npmRegistry) {
+		Bundle bundle, NPMRegistry npmRegistry, JSONObject packageJSONObject,
+		JSONObject packagesJSONObject) {
```

Comment:
What's the difference between `packageJSONObject` and `packagesJSONObject`?

Reply:
- `packageJSONObject` is from `package.json` 
- `packagesJSONObject`is from the `packages` section of `manifest.json`

Maybe I can rename it to `manifestPackagesJSONObject`... :thinking: